tests/jenkins/run_test_amzn_linux_gpu.sh

```diff
@@ -65,5 +65,5 @@ nosetests3 --verbose tests/python/train
 #julia -e 'try Pkg.clone("MXNet"); catch end; Pkg.checkout("MXNet"); Pkg.build("MXNet"); Pkg.test("MXNet")' || exit -1
 
 echo "BUILD scala_test"
-make scalapkg
-make scalatest
+cd scala-package
+mvn install
```

Comment:
Shouldn't this and below be "integration-test" or "integration-test -Dskip.test=false"?